pkgs/applications/networking/misc/zammad/default.nix

```
@@ -151,6 +161,6 @@ stdenv.mkDerivation {
     homepage = "https://zammad.org";
     license = licenses.agpl3Plus;
     platforms = [ "x86_64-linux" "aarch64-linux" ];
-    maintainers = with maintainers; [ n0emis garbas taeer ];
+    maintainers = with maintainers; [ n0emis garbas taeer netali ];
```

```suggestion
    maintainers = with maintainers; [ n0emis taeer netali ];
```

Would you be so kind and also remove me as a maintainer.

---

Done.